### DISCUSSION TITLE
Economic representation of investments

### DISCUSSION THREAD

**Discussion by mihlema** [spine-tools/SpineOpt.jl#864] [Ideas]
In the current implementation, the investment costs appear as a whole at the time of investment.
The investment costs are determined as follows (simplified equation):
```
   investment_costs[s,t] = sum( units_invested[u, s, t] *  unit_investment_cost[(unit=u, t=t)] *
            for (u, s, t) in units_invested_available_indices(m; unit=indices(unit_investment_cost)) 
    )
```
Usually, the investments will not take place in one timestep, but over multiple timesteps (multiple-years). Our current implementation does not properly account for the time value of money. To account for multi-year investments, we might want to  convert our objective function to the net-present value of all (annualized) investment and operational costs. Thoughts @DillonJ @jkiviluo ?

Also, we don't take into account any salvage value at the end of the optimization.

**Comment by TimMertens** [comment]
This document summarizes how I included the economic representation of investment in my planning models up till now. This could serve as a basis for a discussion on how we intend to tackle this issue within Spine. The document is roughly based on the approaches taken within TIMES and the LYSUM invest model. (https://www.mech.kuleuven.be/en/tme/research/energy_environment/Pdf/WP-en2018-07)

[Spine_invest.pdf](https://github.com/Spine-project/SpineOpt.jl/files/6350472/Spine_invest.pdf)

**Comment by clizbe** [comment]
Closing due to stale status.
